### PR TITLE
created new branch with sign up front end

### PR DESCRIPTION
**Pull request by MillieClare**
This has mine and Courtney's front end sign up form with submit. Please review and if good merge with master.

Thanks :)

### CODE REVIEW

File: yuglu-frontend/src/sign-up.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        const target = event.target;
+        const name = target.name;
+        console.log('this is a log', event.target.value);
+        this.setState({
+            [name]: event.target.value
+        });
+    }
+
+    handleSubmit(event) {
+        console.log('A name was submitted: ' + JSON.stringify(this.state));
```
Comment (JoshAdamPowell) [inline yuglu-frontend/src/sign-up.jsx]: remove the console log here

File: yuglu-frontend/src/sign-up.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+export default class SignUpPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleInputChange(event) {
+        const target = event.target;
+        const name = target.name;
+        console.log('this is a log', event.target.value);
```
Comment (JoshAdamPowell) [inline yuglu-frontend/src/sign-up.jsx]: and here

File: yuglu-frontend/src/sign-up.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+export default class SignUpPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleInputChange(event) {
+        const target = event.target;
+        const name = target.name;
```
Comment (JoshAdamPowell) [inline yuglu-frontend/src/sign-up.jsx]: these arent used anymore

File: yuglu-frontend/src/sign-up.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+
```
Comment (JoshAdamPowell) [inline yuglu-frontend/src/sign-up.jsx]: remove extra blank lines

File: yuglu-frontend/src/sign-up.jsx (hunk 3 above)
Comment (MillieClare) [reply yuglu-frontend/src/sign-up.jsx]: Surely const target is being used? I can remove const name tho.

**Pull request merged**